### ISSUE TITLE
Can't create a new team without having provisioned a per-user key

### ISSUE THREAD

**Issue by valleybay** [keybase/client#8528]
I have tried deleting both the iOS and OS X application, both to no avvail.

When logging in after logging out on OS X, I got:
▶ WARNING loginProvision PerUserKeyUpgrade failed: DB error (error 501)

**Comment by maxtaco** [comment]
Please do a 'keybase log send' from the terminal prompt.

**Comment by maxtaco** [comment]
Cc: @mlsteele  can you track this issue?

**Comment by mlsteele** [comment]
Hi @valleybay looks like the server is rejecting your client's attempt to make a PUK. If you do a `keybase log send` I'll be able to find the corresponding server logs.

**Comment by maxtaco** [comment]
I fixed your account on the server, we have to look a bit deeper into why you were affected. I know it has something to do with the fact that you revoked your first key.....

**Comment by valleybay** [comment]
Log submitted with ID be362a7396b11f24ba52de1c

**Comment by jwilkins** [comment]
same issue for keybase user polyclef

**Comment by maxtaco** [comment]
@jwilkins I believe it's fixed in master (correct me if I'm wrong @mlsteele), but you can workaround it right now via `killall keybase`.   Or by turning on your mobile phone.

**Comment by maxtaco** [comment]
@jwilkins actually there's a bug, looking into it.

**Comment by maxtaco** [comment]
OK, another `killall keybase` should fix it now (I hope)

**Comment by maxtaco** [comment]
@jwilkins you should be good; your new key shows up [here](https://keybase.io/polyclef/sigchain#2d27b1116bab2cf0a42b645cd147a2e05f84fab194eb6e07a4489952337ff0af0f)

**Comment by thermistor** [comment]
I also have the error message "ERROR can't create a new team without having provisioned a per-user key"

My log id is d7ef46d91f9ed6811108b61c

**Comment by maxtaco** [comment]
https://keybase.io/thermistor/sigchain#e721bbae1d73646da76943883b1e652f15bcf3792175e025bc09f7201ee6c5560f seems like your should be OK now

**Comment by maxtaco** [comment]
@kylemanna you appear to be logged out. Try `keybase login`?

**Comment by kylemanna** [comment]
Appears I wasn't logged in, oops. I ran `keybase login`.  I checked my `keybase status` before but l glazed over the logged in line.

Now seems to work, thanks!

**Comment by johannorrman** [comment]
Same goes for me.

Log ID: c08024ca18668adadfd0971c

**Comment by kadmil** [comment]
Hey, guys, I have the same thing. `keybase status` shows me logged in

**Comment by maxtaco** [comment]
Do a ‘keybase log send’ and I will take a look.

**Comment by maxtaco** [comment]
@johannorrman seems like when you did log in, it worked, there's just a slight delay.

See [here](https://keybase.io/johannorrman/sigchain#988dd90c79738677f45d1f996a53abe8e1dc4c107cf3a254df5aeed01a48fe310f)

CC: @mlsteele

**Comment by maxtaco** [comment]
@kadmil your bug was fixed on the server-side, it should work as soon as you restart your app, or when the next 1hr-timed background event fires.

**Comment by austinheap** [comment]
@maxtaco if we're stuck with the same issue should we ping support to do some server side magic? (`keybase status` shows me logged in, revoked my first key, etc)

**Comment by maxtaco** [comment]
@austinheap try it again, you should be G2G.  I didn't do anything, you client just had to think about it a bit.

https://keybase.io/austinheap/sigchain#4bb6be99afadfdd33002dc7ca4d5ecbb675f152a8190ab74b5202b602878f4840f

**Comment by austinheap** [comment]
@maxtaco Funny I just noticed that. I quit the desktop client, did a `killall keybase`, reopened the client, and it worked. Thanks for the super fast response!

**Comment by maxtaco** [comment]
yeah cool that would have kicked it into action too.  I believe our next release has a fix for the lag
